stix_shifter_modules/dshield/.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = tests/*
```

Comment:
@SalmanMesia i would move `.coveragerc` to root level. you probably have this in few modules. i would recommend to remove them and put `.coveragerc` in `stix-shifter/`